core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.html

```diff
@@ -1,81 +1,216 @@
-      <span ng-if="editableDescriptionIsEmpty && topicDescriptionChanged" class="form-text" style="font-size: smaller">
+  <div class="topic-editor-left">
+    <a ng-href="/topics-and-skills-dashboard">
+      <button class="btn btn-secondary">Back to Topics Dashboard</button>
+    </a>
+    <md-card class="oppia-subtopics-list-card oppia-long-text">
+      <label class="form-heading">Topic Thumbnail</label>
+      <thumbnail-uploader disabled="!topicRights.canEditTopic()"
+                          filename="topic.getThumbnailFilename()"
+                          update-filename="updateTopicThumbnailFilename"
+                          bg-color="topic.getThumbnailBgColor()"
+                          update-bg-color="updateTopicThumbnailBgColor"
+                          allowed-bg-colors="allowedBgColors"
+                          aspect-ratio="'4:3'"
+                          preview-title="editableName"
+                          preview-description=""
+                          preview-description-bg-color="'#2F6687'"
+                          preview-footer="topic.getCanonicalStoryIds().length + ' Lessons'">
+      </thumbnail-uploader>
+
+      <div class="topic-name">
+        <label for="topicName" class="form-heading protractor-test-topic-name-heading">Topic Name</label>
+        <input id="topicName" type="text" class="form-control protractor-test-topic-name-field"
+               ng-disabled="!topicRights.canEditTopic()"
+               ng-model="editableName" ng-blur="updateTopicName(editableName)"
+               placeholder="Enter a name for the topic." maxlength="<[MAX_CHARS_IN_TOPIC_NAME]>" ng-trim="false">
+        <span class="oppia-input-box-subtitle">
+          <i>
+            Topic name should be at most <[MAX_CHARS_IN_TOPIC_NAME]> characters.
+          </i>
+        </span>
+      </div>
+
+      <div class="topic-description" ng-class="{'has-error': editableDescriptionIsEmpty && topicDescriptionChanged}">
+        <label for="topicDescription" class="form-heading protractor-test-topic-description-heading">Description</label>
+        <textarea type="text" class="form-control protractor-test-topic-description-field"
+                  ng-model="editableDescription" maxlength="<[MAX_CHARS_IN_TOPIC_DESCRIPTION]>"
+                  ng-change="updateTopicDescriptionStatus(editableDescription)"
+                  ng-blur="updateTopicDescription(editableDescription)"
+                  placeholder="Enter the description of the topic"
+                  style="height: 10vh;">
+        </textarea>
+        <span class="oppia-input-box-subtitle">
+          <i>
+            Topic description should be at most <[MAX_CHARS_IN_TOPIC_DESCRIPTION]> characters.
+          </i>
+        </span>
+        <div>
+          <button class="btn btn-secondary" ng-click="togglePreview()">Show/Hide Preview</button>
+          <div ng-if="topic.getId() && topicPreviewCardIsShown">
+            <preview-thumbnail name="editableName"
```

Comment:
Right now, it just shows '1' if there  is only one story in the topic, can you add the suffix as well? Like '1 story'?

Reply:
Used the word Lesson instead of story(I think that's what we use). Let me know.

Reply:
"Story" is the thing that's similar to collection, and "Lesson" is the thing that's similar to exploration. Adjust the terminology accordingly.

Reply:
Ok understood. So Story is suitable here. Done

Reply:
Adjust the word based on the count. It should show 'stories' for 0 and > 1 stories, but it shows just 'story' for everything now.

Reply:
Done.